docker/docker-compose.yml

```diff
@@ -1,36 +1,65 @@
 version: "2"
 services:
+
+  ### 1xx Labs
+  go:
+    build:
+      context: ../go
+    ports:
+      - "7070:80"
+    depends_on:
+      - otelcol
   java:
     build:
       context: ../java
     ports:
       - "8080:80"
     depends_on:
-      - jaeger
+      - otelcol
   node:
     build:
       context: ../node
     ports:
       - "3000:80"
     depends_on:
-      - jaeger
+      - otelcol
```

Comment:
if we have these depend on both `jaeger` and `otelcol`, then we can start with jaeger and move to the collector without having to change the dependencies, which might be a nice thing to have.

Reply:
isn't that kind of how it is with this change? all the apps depend on `otelcol` and `otelcol` depends on `jaeger` and `prometheus`.

Reply:
that will mean we're hitting the collector from the get-go, rather than going straight to jaeger for the first step. I thought we had intended leaving the collector out until 201?